RecoTracker/TrackProducer/python/TrackProducerIterativeDefault_cfi.py

```diff
@@ -1,9 +1,13 @@
 import FWCore.ParameterSet.Config as cms
 import RecoTracker.TrackProducer.TrackProducer_cfi
 
+# for parabolic magnetic field
+from Configuration.ProcessModifiers.trackingParabolicMf_cff import trackingParabolicMf
+
 TrackProducer = RecoTracker.TrackProducer.TrackProducer_cfi.TrackProducer.clone(
```

Comment:
```suggestion
TrackProducerIterativeDefault = RecoTracker.TrackProducer.TrackProducer_cfi.TrackProducer.clone(
```
should've been done a while back. Reusing the same name is pretty dangerous, considering how often `from xyz import *` happen.

Reply:
Done in dedicated commit eaf188f2bd1494bac0be78546b237a4e97835d20.